library/src/main/java/com/fidesmo/fdsm/FidesmoApiClient.java

```diff
@@ -242,4 +242,8 @@ public static String lamei18n(JsonNode n) {
             return n.asText();
         }
     }
+
+    public static boolean isDeveloperMode() {
+        return System.getenv().getOrDefault("FIDESMO_DEVELOPER", "false").equalsIgnoreCase("true");
```

Comment:
IMHO would be better to externalize such configuration parameters (as well as non default API URL) from the library and keep it only in the tool. As it's inconvenient for library users and also would not leak configuration variables from tool to the library.

Reply:
Indeed; will be part of a follow-up that removes the coupling of javax.smartcardio from the library.

Reply:
ok, let's put that into an issue and a separate PR